### DISCUSSION TITLE
[Feature] Ability to favorite partner's photo

### DISCUSSION THREAD

**Discussion by gguglietti** [immich-app/immich#4278] [Feature Request]
### The feature

Currently only owned photos can be tag as favorite. The feature would expand this functionality to assets shared by others. 

In my particular case, my partner and I upload photos from family events, sometimes as part of a shared album, sometimes just sitting on the timeline (accessible to each other). We would both like to be able to favorite photos from each other's timeline for ease of access.

On an additional note, I don't believe the photo should be preserved if the owner / user chooses to delete / stop sharing it in the future. The photo should cease to show up on the favorite in those scenarios.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by jrasm91** [comment]
Duplicate of #2977

**Reply by gguglietti** [reply]
Bug #2977 refers to photos on shared albums. Would the future fix include shared timelines by partners (not part of a shared album)?

**Reply by jrasm91** [reply]
Yeah, it's the same underlying issue - that "is favorite" is tracked at the asset level, and an asset is only owned by one person, thus it can only track the status for a single user. The idea is to migrate to a new format so the favorite status is tracked per user, which would solve it for all places you can favorite an album, including partner and album sharing.

**Reply by gguglietti** [reply]
Makes sense. Thank you!

**Reply by asakusuma** [reply]
https://github.com/immich-app/immich/issues/2977 is closed, but the latest android and web apps won't let me favorite a partner owned photo. on android, there's a warning explicitly saying you can't favorite a partner photo. on web, it just says "added 0 to favorites"

**Comment by kaybi-gh** [comment]
Thank you @martabal for reopeing! 🙏 
The underlying issue is fixed but the feature that would allow to favorite partners assets is not.

**Reply by tbekolay** [reply]
Found this issue from searching. If the underlying fix is in, is the issue a UI one? I.e., it would be possible to do so but the button needs to be added to whatever client (web in my case)?

**Comment by frankyman88** [comment]
Also interested in this!!

**Comment by nullagit** [comment]
Also experiencing this issue with v1.106.4. When I try to favorite a photo uploaded by another account, the UI reports back: "Added 0 to favorites".

**Comment by TKaluza** [comment]
I am also having this issue.

**Comment by phoenixaus** [comment]
Plus one, the option to press the "heart" is not present on images shared but not owned.

**Comment by boredomwontgetus** [comment]
Same for me! This feature would be really great to have. Favorite in shared albums.

**Comment by bo0tzz** [comment]
Folks, as you can tell by all the minimized posts, just saying "same" doesn't add anything to the discussion. Just upvote the original post, please.

**Comment by rjhilgefort** [comment]
Not trying to add another "same" comment but am curious if there's been any movement on this feature in the past couple months. This is the only feature missing that would allow me to ditch Google Photos. Thanks!

**Comment by mcasaccio** [comment]
I am looking for the same functionality

**Reply by bo0tzz** [reply]
Did you see my comment above...?

**Comment by tzhouML** [comment]
Good devs/maintainers, would you be willing to let us know if this is even on your radar? This is a sorely missed feature that I'm sure many of us would love to have.

**Comment by Eric-N-Be** [comment]
Also interested in this!!

Shared albums are often used in families or groups of friends. Being able to keep track of favorite partner photos or tag them (e.g. "vacation 2025", "best of") would improve personal organization

Allow users to add favorites and personal tags on photos shared by partners. These favorites/tags should be only visible to the user who added them, not altering the owner’s data.

**Comment by Lugib** [comment]
upvoted. Really miss this...

**Comment by Eric-N-Be** [comment]
upvoted

**Comment by mcasaccio** [comment]
is there any news about this topic?

**Reply by bo0tzz** [reply]
https://justinmayer.com/posts/any-updates/

**Comment by mcasaccio** [comment]
up

**Comment by mcasaccio** [comment]
I would like to know if there have been any changes to this feature in recent months

**Comment by aryanarora0303** [comment]
Adding a family use case: my partner and I share albums on a self-hosted instance. Neither of us can favourite the other's photos, and we can't see each other's favourites — which feels surprising inside an album that's explicitly shared. Our workaround is maintaining a parallel `<album> favourites` album that we both manually add to, which is extra management for something a per-user favourite flag would solve. Really looking forward to the per-user favourites migration — happy to help test when it lands.

**Comment by freekvh** [comment]
Yes, this would be highly appreciated. Our flow is: Make album, add all photos from all sources (most are already synced from phone), favorite the ones we want to make a physical album from: Download, create album.

But now we are stuck at step 3 and neither of us can make a selection. Perhaps we can make another album add the selection to that... Anyway, would love to have this feature :)
